test_conformance/gl/test_buffers.cpp

```diff
@@ -168,21 +163,21 @@ int test_buffer_kernel(cl_context context, cl_command_queue queue,
     bufferSize = numElements * vecSize * get_explicit_type_size(vecType);
 
     /* Generate some almost-random input data */
-    gen_input_data(vecType, vecSize * numElements, d, inData);
-    memset(outDataCL, 0, dataSize);
-    memset(outDataGL, 0, dataSize);
+    gen_input_data(vecType, vecSize * numElements, d, inData.data());
+    outDataCL.clear();
+    outDataGL.clear();
```

Comment:
One more comment, sorry: `clear` modifies the vector's "size", so I don't think it's what we want to call here.  Note, it doesn't modify the vector's "capacity", and looks like the subsequent part of this test just accesses the vector via `data`, so this "works" but it'd really be better if we didn't modify the vector's size.

https://en.cppreference.com/w/cpp/container/vector/clear

Could we use `assign` or `fill` instead?

https://en.cppreference.com/w/cpp/container/vector/assign
https://en.cppreference.com/w/cpp/algorithm/fill